lib/core/rlocks.nim

```diff
@@ -11,7 +11,10 @@
 
 
 when not compileOption("threads") and not defined(nimdoc):
-  {.error: "Rlocks requires --threads:on option.".}
+  when false:
+    # make rlocks modlue consistent with locks module,
+    # so they can replace each other seamlessly.
+    {.error: "Rlocks requires --threads:on option.".}
```

Comment:
nit: pre-existing but `{.error: "rlocks requires --threads:on option.".}` (lowercase r) would make more sense

Reply:
It is consistent with pre-existing marks
"ThreadPool"
"Locks"
...
I guess they mean the object name instead of the module name.

Reply:
there is not `Rlocks` object name though... it's the whole module that (before this PR) was supposed to be disabled; anyway it's now disabled, and it's a nit